pkg/git/git_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+package git_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
What happened with these imports? They are split up in a very weird way.

Reply:
I have re-order them.